package.json

```diff
@@ -4,40 +4,63 @@
   "repository": "git@github.com:dummyedu/dicomloader.git",
   "author": "dummyedu <dummyedu@gmail.com>",
   "license": "MIT",
+  "main": "lib/dicomloader.js",
+  "unpkg": "dist/dicomloader.js",
+  "module": "es/dicomloader.js",
+  "types": "types/index.d.ts",
+  "files": [
+    "dist",
+    "lib",
+    "es",
+    "src",
+    "types"
+  ],
+  "scripts": {
+    "clean": "rimraf lib dist es coverage types",
+    "format": "prettier --write \"{src,test}/**/*.{js,ts}\" index.d.ts \"**/*.md\"",
+    "format:check": "prettier --list-different \"{src,test}/**/*.{js,ts}\" index.d.ts \"**/*.md\"",
+    "lint": "eslint --ext js,ts src",
+    "check-types": "tsc --noEmit",
+    "test": "jest",
+    "test:watch": "npm test -- --watch",
+    "test:cov": "npm test -- --coverage",
+    "build": "yarn clean && yarn run check-types && yarn run format:check && yarn lint && rollup -c"
```

Comment:
whether yarn clean .... stuff should be put in prebuild.
Please check other open-source project to decide it.